### PR TITLE
Dcv 499 change dag factory so when a dag load fails the rest of the dags are still loaded

### PR DESCRIPTION
**Pull request by BAntonellini**

### CODE REVIEW

File: dagfactory/dagbuilder.py [hunk 1]
```diff
@@ -331,6 +331,18 @@ def set_dependencies(
                     ] = tasks_and_task_groups_instances[dep]
                     source.set_upstream(dep)
 
+    def build_fallback_dag(self, error: Exception) -> Dict[str, Union[str, DAG]]:
```
Comment (ssassi) [inline dagfactory/dagbuilder.py]: why `Dict[str, Union[str, DAG]]` and not just `Dict[str, DAG]`?

Comment (BAntonellini) [reply dagfactory/dagbuilder.py]: I re-used the return signature of the `build` method that's below that one, as they return basically the same

`def build(self) -> Dict[str, Union[str, DAG]]:`

Comment (ssassi) [reply dagfactory/dagbuilder.py]: well, in that one I think it is not accurate as well, dag is always instantiated as a `DAG` instance, not `string`.

File: dagfactory/dagfactory.py [hunk 2]
```diff
@@ -98,10 +99,13 @@ def build_dags(self) -> Dict[str, DAG]:
...
                 if dag:
                     dags[dag["dag_id"]]: DAG = dag["dag"]
-            except Exception as err:
-                raise Exception(
-                    f"Failed to generate dag {dag_name}. verify config is correct"
-                ) from err
+            except Exception:
+                complete_traceback = traceback.format_exc()
+
+                dag: Dict[str, Union[str, DAG]] = dag_builder.build_fallback_dag(
```
Comment (ssassi) [inline dagfactory/dagfactory.py]: Forgot to change type here

**Pull request merged**